docs/development/process.md

```diff
@@ -36,3 +40,48 @@ The validation process for new releases usually takes a couple of days and inclu
 1. The "release responsible" is verifying new features or other notable changes (derived of the draft release notes) in this development system.
 
 If all tests are green, all checks were successful, and the release responsible has performed all of the planned verifications then the release is triggered.
+
+## Cherry Picks
+
+This section explains how to initiate cherry picks on release branches within the `gardener/gardener` repository.
+
+- [Prerequisites](#prerequisites)
+- [Initiate a Cherry Pick](#initiate-a-cherry-pick)
+
+### Prerequisites
+
+Before you initiate a cherry pick, make sure that the following prerequisites are accomplished.
+
+- A pull request merged against the `master` branch.
+- The release branch exists (example: [`release-v1.18`](https://github.com/gardener/gardener/tree/release-v1.18))
+- Have the `gardener/gardener` repository cloned as follows:
+  - the `origin` remote should point to your fork (alternatively this can be overwritten this by passing `FORK_REMOTE=<fork-remote>`)
+  - the `upstream` remote should point to the Gardener github org (alternatively this can be overwritten this by passing `UPSTREAM_REMOTE=<upstream-remote>`)
```

Comment:
```suggestion
  - the `origin` remote should point to your fork (alternatively this can be overwritten by passing `FORK_REMOTE=<fork-remote>`)
  - the `upstream` remote should point to the Gardener github org (alternatively this can be overwritten by passing `UPSTREAM_REMOTE=<upstream-remote>`)
```